src/tests/Feature/Admin/LogTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Tests\Feature\Admin;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\File;
+
+class LogTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testLaravelLogDownloadSuccess()
+    {
+        if (!File::exists(storage_path('logs'))) {
+            File::makeDirectory(storage_path('logs'), 0755, true);
+        }
+        File::put(storage_path('logs/laravel.log'), 'Sample log content');
+        $user = $this->createAdminUser();
+        $token = $user->createToken('test')->plainTextToken;
+        $this->withHeader('Authorization', "Bearer $token")
+            ->get('/api/v1/logs/laravel')
+            ->assertStatus(200)
+            ->assertHeader('Content-Type', 'text/plain; charset=UTF-8')
+            ->assertHeader('Content-Encoding', 'gzip')
+            ->assertHeader('Content-Disposition', 'attachment; filename=laravel.log.gz');
+    }
+
+    public function testLogDownloadNoLogFailure()
```

Comment:
The test name should ideally indicate why it failed. In this case, the name should include NotFound.

Reply:
done